### PR TITLE
updated reflection

### PR DESCRIPTION
**Pull request by meganbradley**

### CODE REVIEW

**Comment by jhutchings1** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[xml/FrameworksIndex/healthvaultclient-1.68.20731.1-preview.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/FrameworksIndex/healthvaultclient-1.68.20731.1-preview.xml) | :white_check_mark:Succeeded | |
[xml/FrameworksIndex/healthvaultnet-2.68.20719.1.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/FrameworksIndex/healthvaultnet-2.68.20719.1.xml) | :white_check_mark:Succeeded | |
[xml/FrameworksIndex/healthvaultweb-1.68.20731.1-preview.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/FrameworksIndex/healthvaultweb-1.68.20731.1-preview.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ApplicationProvisioning/Provisioner.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ApplicationProvisioning/Provisioner.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/ApplicationRecordAuthorizationAction.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/ApplicationRecordAuthorizationAction.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/ApplicationTokenCreationInfo.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/ApplicationTokenCreationInfo.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/AuthenticationTokenCreationStatus.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/AuthenticationTokenCreationStatus.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/CreateAuthenticationTokenResult.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/CreateAuthenticationTokenResult.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/Credential.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/Credential.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/CryptoConfiguration.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/CryptoConfiguration.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/CryptoHash.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/CryptoHash.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/CryptoHashFinalized.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/CryptoHashFinalized.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/CryptoHmacFinalized.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/CryptoHmacFinalized.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Certificate/ApplicationCertificate.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Certificate/ApplicationCertificate.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/CommonNotificationData.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/CommonNotificationData.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/CommonSubscriptionData.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/CommonSubscriptionData.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/HealthRecordItemChangedEventArgs.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/HealthRecordItemChangedEventArgs.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/HealthRecordItemChangedFilter.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/HealthRecordItemChangedFilter.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/HealthRecordItemChangedItem.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/HealthRecordItemChangedItem.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/HealthRecordItemChangedSubscription.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/HealthRecordItemChangedSubscription.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/HttpNotificationChannel.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/HttpNotificationChannel.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/NotificationAuthenticationSharedKeyInfo.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/NotificationAuthenticationSharedKeyInfo.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/NotificationChannel.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/NotificationChannel.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/NotificationHandler.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/NotificationHandler.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/Subscription.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/Subscription.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/SubscriptionManager.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/SubscriptionManager.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/PlanCategory.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/PlanCategory.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/PlanObjective.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/PlanObjective.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/PlanObjectiveState.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/PlanObjectiveState.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/PlanOutcome.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/PlanOutcome.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/PlanOutcomeType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/PlanOutcomeType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/PlanStatus.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/PlanStatus.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskCompletionMetrics.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskCompletionMetrics.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskCompletionType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskCompletionType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskOccurrenceMetrics.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskOccurrenceMetrics.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskRangeMetric.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskRangeMetric.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskRecurrenceType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskRecurrenceType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskSchedule.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskSchedule.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskScheduleType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskScheduleType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskStatus.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskStatus.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTargetEvent.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTargetEvent.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingPolicy.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingPolicy.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingSourceType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingSourceType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingStatus.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingStatus.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingTriggerType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingTriggerType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.Csv/OtherItemDataCsv.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.Csv/OtherItemDataCsv.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.Csv/OtherItemDataCsvItem.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.Csv/OtherItemDataCsvItem.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.Old/AerobicSession.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.Old/AerobicSession.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.Old/AerobicSessionSamples.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.Old/AerobicSessionSamples.xml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C18%5Cd3a7b26e-e832-dfa2-6527-b27014cc5d28%5CPullRequest%5C201708181736174097-39%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: xml/Microsoft.Health.ApplicationProvisioning/Provisioner.xml [hunk 1]
```diff
@@ -77,7 +80,9 @@
       <MemberSignature Language="DocId" Value="M:Microsoft.Health.ApplicationProvisioning.Provisioner.GetApplication(Microsoft.Health.Web.OfflineWebApplicationConnection,System.Guid)" />
       <MemberType>Method</MemberType>
       <AssemblyInfo>
+        <AssemblyName>Microsoft.Health</AssemblyName>
         <AssemblyVersion>2.64.0.0</AssemblyVersion>
```
Comment (davidkwoods) [inline xml/Microsoft.Health.ApplicationProvisioning/Provisioner.xml]: >2.64.0.0 [](start = 25, length = 8)

Will there be any issues keeping the old version in here as well?

File: xml/Microsoft.Health.ItemTypes/GenomeNumberingScheme.xml [hunk 2]
```diff
@@ -1,10 +1,16 @@
-<Type Name="GenomeNumberingScheme" FullName="Microsoft.Health.ItemTypes.GenomeNumberingScheme">
+﻿<Type Name="GenomeNumberingScheme" FullName="Microsoft.Health.ItemTypes.GenomeNumberingScheme">
   <TypeSignature Language="C#" Value="public enum GenomeNumberingScheme" />
   <TypeSignature Language="ILAsm" Value=".class public auto ansi sealed GenomeNumberingScheme extends System.Enum" />
   <TypeSignature Language="DocId" Value="T:Microsoft.Health.ItemTypes.GenomeNumberingScheme" />
   <AssemblyInfo>
     <AssemblyName>Microsoft.Health.ItemTypes</AssemblyName>
     <AssemblyVersion>2.64.0.0</AssemblyVersion>
+    <AssemblyVersion>2.68.0.0</AssemblyVersion>
```
Comment (davidkwoods) [inline xml/Microsoft.Health.ItemTypes/GenomeNumberingScheme.xml]: Not sure what's giving rise to the split/duplicated AssemblyVersion information here...

**Comment by meganbradley** [comment]
Hi @davidkwoods ,
The mdoc reflection tool, OPS, and the API Browser support multiple versions of packages. The packages are merged during mdoc reflection, and the APIs are then filterable by version in API Browser. This is the default way to handle MRef versioning on docs. So when an update request comes in, we add the new versions to the old unless otherwise instructed. If you want to remove old versions, just let me know which you want to keep. I'll follow up on email.

**Comment by davidkwoods** [comment]
@meganbradley What about the ones attributed to multiple assemblies?  GenomeNumberingScheme and a couple others got two AssemblyInfo sections assigning them to more than one assembly, like what's copied below.  Is that because the class was found in both the older .NET SDK and the newer .NET Standard SDK?  If that were the case, I'd expect to see it on almost every ItemType class in there.

  <AssemblyInfo>
    <AssemblyName>Microsoft.Health.ItemTypes</AssemblyName>
    <AssemblyVersion>2.64.0.0</AssemblyVersion>
    <AssemblyVersion>2.68.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>Microsoft.HealthVault</AssemblyName>
    <AssemblyVersion>1.65.0.0</AssemblyVersion>
    <AssemblyVersion>1.66.0.0</AssemblyVersion>
  </AssemblyInfo>

**Comment by jhutchings1** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/meganbradley/healthvault-docs/blob/master/.openpublishing.publish.config.json) | :x:Error | | [Details](#user-content-.openpublishing.publish.config.json)

<a name=".openpublishing.publish.config.json"></a>
### [.openpublishing.publish.config.json](https://github.com/meganbradley/healthvault-docs/blob/master/.openpublishing.publish.config.json)
 - **[Error]** Moniker healthvault-1.66.20706.2-preview doesn&#39;t exist in moniker service. Depot healthvault-dotnet content won&#39;t be published.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C23%5Cd3a7b26e-e832-dfa2-6527-b27014cc5d28%5CPullRequest%5C201708231750184979-39%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jhutchings1** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/meganbradley/healthvault-docs/blob/master/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[xml/FrameworksIndex/healthvaultclient-1.68.20731.1-preview.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/FrameworksIndex/healthvaultclient-1.68.20731.1-preview.xml) | :white_check_mark:Succeeded | |
[xml/FrameworksIndex/healthvaultnet-2.68.20719.1.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/FrameworksIndex/healthvaultnet-2.68.20719.1.xml) | :white_check_mark:Succeeded | |
[xml/FrameworksIndex/healthvaultweb-1.68.20731.1-preview.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/FrameworksIndex/healthvaultweb-1.68.20731.1-preview.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ApplicationProvisioning/Provisioner.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ApplicationProvisioning/Provisioner.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/ApplicationRecordAuthorizationAction.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/ApplicationRecordAuthorizationAction.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/ApplicationTokenCreationInfo.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/ApplicationTokenCreationInfo.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/AuthenticationTokenCreationStatus.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/AuthenticationTokenCreationStatus.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/CreateAuthenticationTokenResult.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/CreateAuthenticationTokenResult.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/Credential.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/Credential.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/CryptoConfiguration.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/CryptoConfiguration.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/CryptoHash.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/CryptoHash.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/CryptoHashFinalized.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/CryptoHashFinalized.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Authentication/CryptoHmacFinalized.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Authentication/CryptoHmacFinalized.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Certificate/ApplicationCertificate.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Certificate/ApplicationCertificate.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/CommonNotificationData.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/CommonNotificationData.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/CommonSubscriptionData.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/CommonSubscriptionData.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/HealthRecordItemChangedEventArgs.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/HealthRecordItemChangedEventArgs.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/HealthRecordItemChangedFilter.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/HealthRecordItemChangedFilter.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/HealthRecordItemChangedItem.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/HealthRecordItemChangedItem.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/HealthRecordItemChangedSubscription.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/HealthRecordItemChangedSubscription.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/HttpNotificationChannel.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/HttpNotificationChannel.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/NotificationAuthenticationSharedKeyInfo.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/NotificationAuthenticationSharedKeyInfo.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/NotificationChannel.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/NotificationChannel.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/NotificationHandler.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/NotificationHandler.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/Subscription.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/Subscription.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.Events/SubscriptionManager.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.Events/SubscriptionManager.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/PlanCategory.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/PlanCategory.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/PlanObjective.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/PlanObjective.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/PlanObjectiveState.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/PlanObjectiveState.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/PlanOutcome.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/PlanOutcome.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/PlanOutcomeType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/PlanOutcomeType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/PlanStatus.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/PlanStatus.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskCompletionMetrics.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskCompletionMetrics.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskCompletionType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskCompletionType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskOccurrenceMetrics.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskOccurrenceMetrics.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskRangeMetric.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskRangeMetric.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskRecurrenceType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskRecurrenceType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskSchedule.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskSchedule.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskScheduleType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskScheduleType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskStatus.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskStatus.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTargetEvent.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTargetEvent.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingPolicy.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingPolicy.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingSourceType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingSourceType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingStatus.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingStatus.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingTriggerType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingTriggerType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingType.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.ActionPlans/TaskTrackingType.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.Csv/OtherItemDataCsv.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.Csv/OtherItemDataCsv.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.Csv/OtherItemDataCsvItem.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.Csv/OtherItemDataCsvItem.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.Health.ItemTypes.Old/AerobicSession.xml](https://github.com/meganbradley/healthvault-docs/blob/master/xml/Microsoft.Health.ItemTypes.Old/AerobicSession.xml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C23%5Cd3a7b26e-e832-dfa2-6527-b27014cc5d28%5CPullRequest%5C201708231801001132-39%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**